samples/discord-guild-chat/extension/index.ts

```diff
@@ -0,0 +1,31 @@
+import { NodeCG } from "nodecg/types/server";
+import { ServiceProvider } from "nodecg-io-core/extension/types";
+import { DiscordServiceClient } from "nodecg-io-discord/extension";
+
+
+module.exports = function (nodecg: NodeCG) {
+    nodecg.log.info("Sample bundle for discord started");
+
+    // This implicit cast determines the client type in the requireService call
+    const discord: ServiceProvider<DiscordServiceClient> | undefined = nodecg.extensions["nodecg-io-discord"] as any;
+
+
+    discord?.requireService("Discord-guild-chat", (client) => {
```

Comment:
Developer UX problem: these bundle names should be the same as the ones in the `package.json` of the corresponding bundle. Maybe we should require the nodecg instance instead, which contains a variable with the bundle name. That way developers can't mess this up.
```suggestion
    discord?.requireService("discord-guild-chat", (client) => {
```